BE/src/main/java/yanse/airbnb/domain/member/Members.java

```diff
@@ -10,24 +10,34 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.OneToMany;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
 import yanse.airbnb.domain.reservation.Reservation;
-import yanse.airbnb.domain.wish.WishList;
-
+import yanse.airbnb.domain.wish.Wish;
+@Getter
+@Builder
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
 @Entity
-public class Member {
+public class Members {
```

Comment:
예약어 이슈인가요? 이런 방식으로 사용할 수 있긴 합니다.

```suggestion
@Table(name = "members")
public class Member {
```

Reply:
오 감사합니다 :) 다음부터 예약어 이슈가있을때 사용하도록하겠습니다 :)